### PR TITLE
XAudio2: load up 2.9 dll if available

### PR DESCRIPTION
**Pull request by raven02**

### CODE REVIEW

File: rpcs3/Emu/Audio/XAudio2/XAudio2Thread.cpp [hunk 1]
```diff
@@ -4,14 +4,14 @@
 #include "Utilities/StrFmt.h"
 #include "Utilities/Config.h"
 #include "Emu/System.h"
-
+#include "stdafx.h"
```
Comment (Nekotekina) [inline rpcs3/Emu/Audio/XAudio2/XAudio2Thread.cpp]: not this...

File: rpcs3/Emu/Audio/XAudio2/XAudio2Thread.cpp [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
 #include "XAudio2Thread.h"
 #include <Windows.h>
 
 extern cfg::bool_entry g_cfg_audio_convert_to_u16;
 
 XAudio2Thread::XAudio2Thread()
-	: m_xaudio(LoadLibraryA("xaudio2_8.dll"))
+	: m_xaudio(fs::is_file("C:/Windows/System32/XAudio2_9.dll") ? LoadLibraryA("xaudio2_9.dll") : LoadLibraryA("xaudio2_8.dll"))
```
Comment (Nekotekina) [inline rpcs3/Emu/Audio/XAudio2/XAudio2Thread.cpp]: Looking to C:/Windows/... is absolutely wrong... Just checking result of LoadLibraryA is enough

**Comment by Nekotekina** [comment]
If 2.8 is broken it may be better to prioritize 2.7 over 2.8 but 2.7 is slightly harder to detect since it's not loaded from DLL directly.

**Comment by raven02** [comment]
That would be good as well to prioritize 2.7 over 2.8 .I'm closing this one .

**Comment by danilaml** [comment]
@raven02 what's wrong with 2.8?

**Comment by raven02** [comment]
@danilaml , i found the 2.8 dll is very small when compared to the 2.7 and 2.9 . And , sometimes it failed to AddData() . kd-11 also confirm this issue.

**Comment by danilaml** [comment]
@raven02 2_7 fails to add data sometimes too. Very frequent when using PPU recompiler.

**Comment by Nezarn** [comment]
@danilaml but this issue started after #1961 (at least for me)

**Comment by danilaml** [comment]
@raven02 It's 11KB most likely because, as Nekotekina said, it's only an "interface" for 2_9 and uses it internally. They very similar.

**Pull request closed without merging**